cmd/canary/main.go

```diff
@@ -60,6 +66,26 @@ func startHandler(c *cli.Context) {
 	if err := canary.Run(); err != nil {
 		log.Fatal("Failed to run canary: ", err)
 	}
+
+	dumpMemoryProfile(c)
```

Comment:
Do we need this now? We are already registering pprof handler.

Reply:
I would keep it. Still like it better.